chain/network/src/routing/routing_table_actor.rs

```diff
@@ -392,6 +408,11 @@ impl Actor for RoutingTableActor {
     type Context = Context<Self>;
 
     fn started(&mut self, _ctx: &mut Self::Context) {}
+
+    /// Try to gracefully disconnect from active peers.
```

Comment:
what active peers?

Reply:
"active peers" in this case means peers we are connected to that are in "active_peers" struct.

We should rename it.
https://github.com/near/nearcore/issues/5424